packages/discord.js/src/util/Util.js

```diff
@@ -415,26 +446,29 @@ function cleanContent(str, channel) {
 
 /**
  * The content to put in a code block with all code block fences replaced by the equivalent backticks.
+ *
  * @param {string} text The string to be converted
  * @returns {string}
  */
 function cleanCodeBlockContent(text) {
-  return text.replaceAll('```', '`\u200b``');
+  return text.replaceAll('```', '`\u200B``');
 }
 
 /**
  * Parses a webhook URL for the id and token.
+ *
  * @param {string} url The URL to parse
  * @returns {?WebhookClientDataIdWithToken} `null` if the URL is invalid, otherwise the id and the token
  */
 function parseWebhookURL(url) {
-  const matches = url.match(
-    /https?:\/\/(?:ptb\.|canary\.)?discord\.com\/api(?:\/v\d{1,2})?\/webhooks\/(\d{17,19})\/([\w-]{68})/i,
-  );
+  const matches =
+    /https?:\/\/(?:ptb\.|canary\.)?discord\.com\/api(?:\/v\d{1,2})?\/webhooks\/(?<id>\d{17,19})\/(?<token>[\w-]{68})/i.exec(
+      url,
+    );
 
-  if (!matches || matches.length <= 2) return null;
+  if (!matches?.groups) return null;
```

Comment:
`groups` will always be an object if there are groups in the regex. Consistency with the check being done in `parseEmoji`

```suggestion
  if (!matches) return null;
```